src/analysis/functions.rs

```diff
@@ -739,11 +732,8 @@ fn analyze_function(
 
     if r#async && status.need_generate() && !commented {
         if env.config.library_name != "Gio" {
-            imports.add("gio_sys");
             imports.add_with_constraint("gio", version, None);
```

Comment:
Can be removed, can't it? Check if there are more `gio` imports being added elsewhere :)

Reply:
Oh good point! More imports removed! \o/